pkg/runners/zero/zero.go

```diff
@@ -303,11 +303,17 @@ func (runner *ZeroRunner) checkRangeCheckUsage() error {
 	var rcUnitsUsedByBuiltins uint64
 	for _, builtin := range runner.program.Builtins {
 		if builtin == starknet.RangeCheck {
-			bRunner := builtins.Runner(builtin)
-			rangeCheckRunner, _ := bRunner.(*builtins.RangeCheck)
-			rangeCheckSegment, ok := runner.vm.Memory.FindSegmentWithBuiltin(rangeCheckRunner.String())
-			if ok {
-				rcUnitsUsedByBuiltins += rangeCheckSegment.Len() * rangeCheckRunner.RangeCheckNParts
+			for _, layoutBuiltin := range runner.layout.Builtins {
```

Comment:
nice indeed!

Reply:
what was the bug related to this? 

i just see that a file with rangecheck usage. but with a layout plain could successfully execute this function?

Reply:
Only faced an issue with the blake hash test.

Reply:
oh ok